packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -82,6 +84,10 @@ export interface CaptionsInfo {
    * Timestamp of when the captioned words were initially spoken.
    */
   timestamp: Date;
+  /**
+   * Timestamp of when the captions were last updated.
+   */
+  lastUpdatedTimestamp?: Date;
```

Comment:
This could be breaking for stateful customers, 
maybe we can change the comments and add a new prop createdTimeStamp